semgrep/semgrep/cli.py

```diff
@@ -533,6 +533,10 @@ def cli(
             version_check()
         return
 
+    # Creates a dict mapping all local variables to their arugments
+    # Because no other variables have been defined yet, this contains only the arguments to the function
+    user_flags = locals()
```

Comment:
I think we're more interested in what user flags were used and less what values they eventually get in run time. So we might want to collect this info one level up?

Reply:
We probably also don't care about what targets user is running on?

Reply:
I think we really only want flags, not options with finite arity. Or at least, we want to be really careful with these, as they could contain customer-sensitive data we'd want to avoid collecting.

Reply:
I'd suggest using an allow-list of options we want to collect, to ensure we don't accidentally collect a sensitive option now or in the future.

Reply:
Also, you'll need to update PRIVACY.md with the additional collected values (together with _why_ we're collecting those data)